test/jllenvs.jl

```diff
@@ -0,0 +1,9 @@
+@testset "get_default_args for all triples" begin
+    # just testing this does not crash on any default triple
+    Clang.get_default_args.(Clang.JLLEnvs.JLL_ENV_TRIPLES)
+end
```

Comment:
this test downloads artifacts for all platforms. running it every time is a bit overkill. 

is there a way to somehow cache it?

Reply:
or maybe we can control it by env variables.

Reply:
sorry i didn't realize it did that, i've simplified the test